steps/pytorch/validation.py

```diff
@@ -6,13 +6,16 @@
 
 
 class DiceLoss(nn.Module):
-    def __init__(self):
+    def __init__(self, smooth=0, eps = 1e-7):
```

Comment:
@apyskir @taraspiotr 
Can we go for Softmax in the first iteration? 
I do want to experiment with Sigmoid here but let's start simple (without mixing softmax in CE and sigmoid in DICE). Am I missing something?